src/Toolkit/Toolkit.UI.Controls/FloorFilter/FloorFilter.cs

```diff
@@ -658,7 +659,20 @@ public void SetSelectedLevelWithoutZoom(FloorLevel newLevel)
         /// <remarks>
         /// This is used for showing an entire facility in 3D.
         /// </remarks>
+        [Obsolete("This property is deprecated. Use AllDisplayLevelsSelected instead.")]
         public bool AllDisplayLevelsSelecteded
```

Comment:
I would add `[EditorBrowsable(EditorBrowsableState.Never)]` to hide this from IntelliSense